Makefile

```diff
@@ -1,5 +1,5 @@
 MOD_FLAGS := $(shell (go version | grep -q -E "1\.(11|12)") && echo -mod=vendor)
-CMDS  := $(addprefix bin/, $(shell go list $(MOD_FLAGS) ./cmd/... | xargs -I{} basename {}))
+CMDS  := $(addprefix bin/, $(shell go list $(MOD_FLAGS) ./cmd/... | xargs -I{} basename {} | grep -Fv "cli"))
```

Comment:
Nit: I think it'd be better to at least use "cmd/cli" here, but it's probably fine.

Reply:
Oops, I didn't think about the xargs here. Ignore me.